docs/src/modules/concepts/pages/architecture-model.adoc

```diff
@@ -64,9 +64,28 @@ The API layer may also expose public event models over Kafka or other channels.
 
 Access control and request validation also belong here. For HTTP-specific guidance, see xref:java:http-endpoints.adoc[Designing HTTP Endpoints].
 
-== Akka Services
+== Akka services
 
-image:ROOT:service.png[Services,width=100,float=left] A _Project_ may contain multiple _Services_. Projects can be deployed to one or more regions to achieve geographic resilience. For details, see xref:multi-region.adoc[].
+image:ROOT:service.png[Services,width=50,float=left] A _Project_ may contain multiple _Services_. Projects can be deployed to one or more regions to achieve geographic resilience. For details, see xref:multi-region.adoc[]. Services are a hard boundary and unit of deployment in Akka projects.
+
+=== Multi-service sharing
+
+Code and components can also be shared between Akka services. There are a few guidelines to remember when setting up this kind of sharing in large, multi-service projects.
+
+If you're following the recommended project structure, then your code should fall into one of three categories: _API_, _Application_, and _Domain_. 
+
+Below is a list of some considerations you should keep in mind when sharing.
+
+* ⚠️  **Entities** - Shared entities can make it easy for multiple services to agree on a set of events and commands. Be careful to avoid multiple, isolated entities sharing the same _internal_ state. Keep in mind that the same entity in different services will not share data. Akka will consider them two different entities with regard to persistence.
```

Comment:
One other way of using shared libraries, is to use DTOs. A service that produces events, for instance, could have a DTO library that is shared to consumers. 

When streaming out, the events are transformed to the common DTO format and easily consumed on the other side. 

This kind of sharing do not evolve the components themselves.